scripts/exsrw_smoke_dust.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+#
+#-----------------------------------------------------------------------
+#
+# Source the variable definitions file and the bash utility functions.
+#
+#-----------------------------------------------------------------------
+#
+. ${USHsrw}/source_util_funcs.sh
```

Comment:
Please replace `USHsrw` with `USHdir`.
```suggestion
. $USHdir/source_util_funcs.sh
```